spec/overview.html

```diff
@@ -73,6 +73,14 @@ <h1>Implementation Dependencies</h1>
       </li>
     </ul>
 
+    <emu-note>
+      Changes to the set of locales, currencies, calendars, numbering systems, and other enumerable items made available by ECMAScript hosts must not result in two users using the same version of the same host on the same platform becoming distinguishable from each other. This constraint is imposed to reduce the fingerprinting risk inherent in internationalization, and may be relaxed in future revisions.
```

Comment:
Much clearer, will use this

Reply:
The notion of "user agent string" is foreign to ECMAScript and not a universal among implementations. @ljharb, please take a look, it seems to be going contrary to your comments above.

As something far from matters typically expected to be specified in a programming language (or API closely related to it, being an optional extension of a language's standard library) and pertaining to only a subset of implementations, namely Web browsers, shouldn't it live in a spec for those? Like the one defining what a Web browser is and what requirements it has to satisfy as an ECMAScript implementation (if supported) in addition to ECMA-262?

Reply:
I agree the concept and mention of a user agent doesn’t make any sense in an ecma specification.

Reply:
In that case I think we need to step back and consider the premise here. The fingerprinting concerns are largely only relevant for web browser environments. 

_Either_ it is ECMA402's job to address these fingerprinting concerns and thus it _must_ be allowed to refer to mechanisms available in those contexts, _or_ it is not ECMA402's job, and we don't need to handle this at all. We can't have our fingerprinting cake if we're not planning on eating it.

I suspect the framing here can be refined a bit to be clear that it is talking in a web browser context only. Alternatively, a more general point can be made about "systems where fingerprinting is a concern, like web browsers", and instead of saying UA strings talk about "already distinguishable bits of information (in the case of browsers, this is platform/version/UA string)". Something like that.

Reply:
IMO it really belongs elsewhere. But if a critical mass insists it's desirable to make the spec longer and draw the attention also therein to this concern present in some implementations, an **informative** note with sufficiently generic wording could be added.